### DISCUSSION TITLE
[BUG] workflow_dispatch not triggering from a GitHub action

### DISCUSSION THREAD

**Discussion by andersson09** [community/community#26790] [Actions]
<p>Hi,</p>
<p>-Manual REST API call from Postman works fine with successful 204 response.<br>
-REST call from another GitHub workflow does not even when using the same curl and also 204 response.</p>
<p>Not sure how to trigger the manual dispatch via another github workflow. I’ve tried the marketplace action and it also does not work.</p>
<p>Thanks</p>

**Comment by Simran-B** [comment, marked as answer]
<p>If you want to trigger a workflow from another workflow, you need to use a personal access token. The default token does not allow workflows to invoke other workflows to prevent recursive calls.</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#triggering-new-workflows-using-a-personal-access-token">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">

      <a href="https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#triggering-new-workflows-using-a-personal-access-token" target="_blank" rel="noopener nofollow ugc">GitHub Docs</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/7819991/181099924-fb57e2fe-280a-4ce7-970b-a6678674e948.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#triggering-new-workflows-using-a-personal-access-token" target="_blank" rel="noopener nofollow ugc">Events that trigger workflows - GitHub Docs</a></h3>



  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

**Comment by andersson09** [comment]
<p>Thanks it works! I must have missed this somewhere.</p>
